src/scripts/main.js

```diff
@@ -1,3 +1,10 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+spider.style.left = `${(parseFloat(wall.clientWidth) / 2)
+- (parseFloat(spider.clientWidth) / 2)}px`;
+
+spider.style.top = `${(parseFloat(wall.clientHeight) / 2)
+- (parseFloat(spider.clientHeight) / 2)}px`;
```

Comment:
parseFloat redundant.

```suggestion
spider.style.left = `${wall.clientWidth / 2 - spider.clientWidth / 2}px`;
spider.style.top = `${wall.clientHeight / 2 - spider.clientHeight / 2}px`;
```